### ISSUE TITLE
Changes to Accessibility Checker sample

### ISSUE THREAD

**Issue by mlewand** [ckeditor/ckeditor-sdk#182]
In general it looks very good. Good job with incorporating new content into a sample. :)

Here are couple of things that came out while I was checking the sample:

## General

1. We need to exhibit more the AC button. Let's put it together with a source button as a very first group in the toolbar.
1. There is no `<script>` elem, fetching the jQuery in code listing.
1. We should remove spell checker button.

## Sample content

* "stranger to proper web standards", I'd suggest going simply with "stranger to web standards"
* order of checklists should be changed to reflect the following:
	* WCAG2
	* Section508
	* WAI-ARIA
	* IBM WAC

**Comment by AnnaTomanek** [comment]
Re point 2 above - this is how the sample source code page is generated. It seems we need a way to be able to mark some additional elements (like required scripts, stylesheets perhaps, too?) as needed in the source version - @Tade0, any ideas?

**Comment by mlewand** [comment]
By the look of it, I guess it comes down to adding `data-sample="1"` to element that should be included into listing.

```
<script data-sample="1" src="../vendor/jquery/jquery-1.12.3.min.js"></script>
```

The problem here is that it's not a copy'n'paste friendly code. However if we'd link jQuery to their cdn, it would solve the portability issue. So I'm proposing simply to change it into:

```
<script data-sample="1" src="https://code.jquery.com/jquery-1.12.3.min.js"></script>`
```

After that we should check if `vendor/jquery/*` is used in any other place, if not it can be removed from the repo.
